app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -265,6 +267,29 @@ a.inplace-editing--trigger-link,
     &.edit-strategy-textarea
       margin-top: auto
 
+// Explicit styles for comment / input-esque trigger appearance
+.work-packages--activity--add-comment
+  margin: 20px 0
+
+  .inplace-editing--container
+    border: 1px solid #eee
+
+  .inplace-edit--read-value
+    padding: 10px 5px
+
+  .inplace-editing--trigger-link
+    color: #888
+    font-size: 1rem
```

Comment:
In my opinion the fields are different because one is / gets a text area and the other is a select field. We should return to the question how we want to display those fields later. For watchers and comments I would directly show the form fields.